decentraland/unity-explorer
# Fix: LiveKit sync issues and upgrade

Fixes #1632 
Fixes #1661 
Fixes #1627 

**(Not fixed)**
 #1633 
 #1631 

- Upgraded LiveKit SDK to https://github.com/livekit/rust-sdks/releases/tag/ffi-v0.7.2 
- Fixed bugs with `Send` and `SendBinary` regarding `comms` messages

**How to QA:**
- As the LiveKit SDK was upgraded a thorough test of all multiplayer capabilities is required
- Test the scenes from the tickets
- Regarding #1661
   - it's a very specific issue that is not connected to any scene
   - try any goerli scene as it's very isolated from everything else and from Archipelago
   - if you spot that according to the connected users stats in the room section does not match the number of users you see, it's a signal of the problem
   - it may need several connections/disconnections to the same scene to reproduce it
   - it was easily reproducible just with a pair of users, no need for a big group. It could happen that on one end both users are seen while on the other a peer is not seen at all
